pkg/manager/recorder.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Is this just a copied module or do we have our own changes here?

Reply:
just copy